UnitTests/GitCommands.Tests/AppTitleGeneratorTests.cs

```diff
@@ -16,10 +18,17 @@ public class AppTitleGeneratorTests
         [SetUp]
         public void Setup()
         {
+            // Create the test instance
+            _appTitleGenerator = new();
```

Comment:
Think we need to use full name of class istead `new()`.

Reply:
In cases like this (and in non-prod code) I dont think it particular matters. 

That is, "_new up the known instance_" vs "_get a response of <?> type by calling some method_".
```cs
private AppTitleGenerator _appTitleGenerator;

// Is it really that much more (un)readable?
 _appTitleGenerator = new();
 _appTitleGenerator = new AppTitleGenerator();
```
vs
```cs
var appTitleGenerator = someType.ExecuteSomeMethod();
AppTitleGenerator appTitleGenerator = someType.ExecuteSomeMethod();
```

Reply:
I am proceeding from line-by-line reading, and the type must occur once in a line.
Otherwise I don't understand when we use it.